packages/jest-transform/src/types.ts

```diff
@@ -38,6 +37,7 @@ export type TransformedSource = {
 
 export type TransformResult = {
   script: Script;
+  scriptContent: string;
```

Comment:
this is not strictly needed now, but is for #9252. I can remove it if we want, but I think it makes sense to return the transformed code _not_ wrapped in a `vm.Script` to make it more generally usable